tawkto/tawkto.php

```diff
@@ -92,6 +92,10 @@ public static function init_options() {
 			if ( ! get_option( self::TAWK_SECURITY_OPTIONS, false ) ) {
 				update_option( self::TAWK_SECURITY_OPTIONS, $options['security'] );
 			}
+
+			if ( ! get_option( self::TAWK_CONFIG_VERSION, false ) ) {
```

Comment:
Why fetch config with `false` as the fallback? IMHO - simple strict comparison with `null` instead of `!` will make it more straight-forward and remove the type ambiguity